imagej/imagej-ops2/src/main/java/net/imagej/ops2/help/HelpForName.java

```diff
@@ -31,26 +31,35 @@
 
 import java.util.function.BiFunction;
 
+import org.scijava.ops.api.OpEnvironment;
 import org.scijava.ops.api.OpInfo;
+<<<<<<< HEAD
```

Comment:
Note the conflicts embedded in this commit. I did this so that they can be subsequently fixed, and the commit amended, while establishing the proper new base on top of the current main branch.